lib/network/modules/EdgesHandler.js

```diff
@@ -108,7 +108,7 @@ class EdgesHandler {
       value: undefined
     };
 
-    util.extend(this.options, this.defaultOptions);
+    util.deepExtend(this.options, this.defaultOptions);
```

Comment:
Beats me; I only knew `lodash` by name, not what it does. It might be worth investigating, and perhaps if relevant open a new issue for conversion.

I just used the functionality that is present.